tests/unit/s2n_override_openssl_random_test.c

```diff
@@ -26,7 +28,7 @@
 #include "utils/s2n_random.h"
 #include "utils/s2n_safety.h"
 
-#if S2N_LIBCRYPTO_SUPPORTS_CUSTOM_RAND
+#if S2N_LIBCRYPTO_SUPPORTS_ENGINE
```

Comment:
What part of this test doesn't compile without engine support? What methods are you gating here? Are they present in the feature probe?